packages/form-js-viewer/src/Form.js

```diff
@@ -444,39 +482,87 @@ export default class Form {
   /**
    * @internal
    */
-  _initializeFieldData(data) {
-    const formFieldRegistry = this.get('formFieldRegistry'),
-          formFields = this.get('formFields'),
-          pathRegistry = this.get('pathRegistry');
+  _getInitializedFieldData(data, options = {}) {
+    const formFieldRegistry = this.get('formFieldRegistry');
+    const formFields = this.get('formFields');
+    const pathRegistry = this.get('pathRegistry');
 
-    return formFieldRegistry.getAll().reduce((initializedData, formField) => {
-      const {
-        defaultValue,
-        type
-      } = formField;
-
-      // try to get value from data
-      // if unavailable - try to get default value from form field
-      // if unavailable - get empty value from form field
+    const {
+      customRoot,
+      customIndexes
```

Comment:
Maybe calling them simple `container` and `indexes`? The fact these are custom is clear as they are optional attributes.